tools/tune/drc/testbench_common/get_drc_mag2db_fixed.m

```diff
@@ -26,24 +24,25 @@
     %	column2: double (%f)
     %   column3: double (%f)
     %	column4: categorical (%C)
+    % For more information, see the TEXTSCAN documentation.
     formatSpec = '%11f%12f%15f%C%[^\n\r]';
 
     %% Open the text file.
-    fileID = fopen(fullFileName,'r');
+    fileID = fopen(filename,'r');
 
     %% Read columns of data according to the format.
-    % This call is based on the structure of the file.
     dataArray = textscan(fileID, formatSpec, 'Delimiter', '', 'WhiteSpace', '', 'TextType', 'string', 'HeaderLines' ,startRow-1, 'ReturnOnError', false, 'EndOfLine', '\r\n');
 
     %% Close the text file.
     fclose(fileID);
 
+
     %% Create output variable
-    file = table;
-    file.idx = dataArray{:, 1};
-    file.testvector = dataArray{:, 2};
-    file.Fixlog10linear = dataArray{:, 3};
-    file.VarName4 = dataArray{:, 4};
+    mag2dB = table;
```

Comment:
should unify the name to "lin2db", not "mag2db" or "lin2dB"

Reply:
Standard nomenclature mag2db means - magnitude data Y into dB values.